Documentation/ContentObjects/UserAndUserInt/Index.rst

```diff
@@ -139,16 +139,17 @@ from TypoScript. Use this TypoScript configuration:
    page = PAGE
    page.10 = USER_INT
    page.10 {
-     userFunc = Vendor\ExtensionName\ExampleTime->printTime
+     userFunc = Vendor\SitePackage\UserFunctions\ExampleTime->printTime
    }
 
-The file typo3conf/ext/extension_name/Classes/ExampleTime.php might amongst other things
-contain:
+The file :file:`EXT:site_package/Classes/UserFunctions/ExampleTime.php` might
+amongst other things contain:
 
 .. code-block:: php
```

Comment:
```suggestion
.. code-block:: php
   :caption:  EXT:site_package/Classes/UserFunctions/ExampleTime.php
```

Reply:
Yes, also thought about it but wanted to avoid doubling from the text above. However, it does not hurt ;-)

Reply:
I sometimes only use the file name in the text, makes it easier to read and then the complete name in the caption